_layouts/project.html

```diff
@@ -63,7 +63,7 @@ <h6 class="contributor">{{ page.contributor }}</h6>
     <div id="{{ page.volume }}{{ page.pages }}" class="citation modal">
       <div class="modal-content">
         <h4>Citation</h4>
-        <p>{{ page.contributor }}. {{ page.title }}. In Emergency Index Vol. {{ page.volume }}, Brooklyn NY: Ugly Duckling Presse, {{ page.volume }}, {{ page.pages }}. Accessed on {{ site.time | date: '%B %d, %Y' }} from <span class="location-origin"></span>{{ page.url }}.</p>
+        <p>{{ page.contributor }}. {{ page.title }}. In Emergency Index Vol. {{ page.volume | slice: -1 }}, Brooklyn NY: Ugly Duckling Presse, {{ page.volume }}, {{ page.pages }}. Accessed on {{ site.time | date: '%B %d, %Y' }} from <span class="location-origin"></span>{{ page.url }}.</p>
```

Comment:
not quite sure what the `| slice: -1` is doing here?

also, isn't this the wrong place to change this? the Emergency Index Vol. would still be the value of `{{page.volume}}`. don't we want to change the next occurrence? `Brooklyn NY: Ugly Duckling Presse, {{ page.volume }}` and couldn't you use `{{ page.volume | plus: 1}}`?

Reply:
also, is the "Accessed On" date needed here? wouldn't that always be right now? 🤔

Reply:
So the problem as Yelena outlined it was not what I've seen.

For example, when I click the citation button, I get: "EDGAR FABIAN FRIAS. GIVE US GODDESS WOMB. In Emergency Index Vol. 2016, Brooklyn NY: Ugly Duckling Presse, 2016, 324-325. Accessed on January 17, 2020 from https://emergencyindex.com/projects/2016/324-325."

Note this part "In Emergency Index Vol. 2016"

That is WRONG - the Volume NUMBER should be there, not the year.

We can easily get the Volume number by using 'slice -1' to get the 6 from 2016.

----

RE: Accessed On date, that IS always right now, and whenever the person clicks that citation button it will give them the date which they accessed that page on, which would always be whenever they click it. Which would always be "right now"

Reply:
@whatcoloris ah, the single digit number for the volume and the `slice: -1` makes sense. and i get your "sustainable" comment more now. because yeah, after 2021 this logic won't work anymore. let's try to find a better way. 

but isn't the year + 1 thing needed where the second `{{page.volume}}` is used? does that make any sense? 

also, the accessed on date. i guess i assume you're talking about an ideal state, because it's not showing right meow.

Reply:
one way to get from year -> vol. number would to be just subtract `2010` from the volume so `{{ page.volume | minus: 2010 }}`?

Reply:
this is all great. i'm stupid about the "now" - updated the page, things "should" work now.

Reply:
cool cool. i don't think we can use ruby/liquid date functions here because "now" would be when the page is built and not when the user actually views it? think we'd need to use javascript for that. i'll try to put something together.

Reply:
AH good point about the "now" - you're absolutely right - everything else works - HOORAY!